asmcli/commands/validate.sh

```diff
@@ -91,15 +91,32 @@ validate_dependencies() {
 }
 
 validate_control_plane() {
-  if is_managed; then
+  if is_autopilot; then
+    validate_autopilot
+  elif is_managed; then
```

Comment:
Is there a reason this is an `elif` instead of two separate if blocks?

Reply:
I think they have the same effect. If it's not managed or legacy, it would've aborted, so it wouldn't reach the next if. I think `elif` makes it clear that it's either Autopilot or legacy MCP checks. I could make it two separate if blocks if you prefer?

Reply:
Nope, that make sense. Just wanted to make sure I understood.